docs/news.d/preemptive_acknacks.rst

```diff
@@ -0,0 +1,6 @@
+.. news-prs: 4735
+
+.. news-start-section: Additions
+- OpenDDS now sends preemptive (preassociation) acknacks in a form that RTI Connext understands.
+- Preassociation acknacks now falloff like heartbeats.
```

Comment:
```suggestion
- Preassociation acknacks now fall off like heartbeats.
```

Should Preassociation be Pre-association?

Reply:
I think so.